src/components/MediaSettings/MediaSettings.vue

```diff
@@ -561,7 +562,6 @@ export default {
 				emit('local-video-control-button:toggle-video')
 			}
 
-			this.updatePreferences()
```

Comment:
If I join the call first time (or at least without these values in storage), and some device are selected which I'm fine with, I'd expect them to be preferred anyway. So maybe let's keep preferences update here?
If you select somethiing manually, new update from this place just'll make an early return 
```
this.updatePreferences('audioinput')
this.updatePreferences('videoinput')
```

Reply:
good point